site/docs/foundations/size.mdx

```diff
@@ -5,11 +5,11 @@ sidebar:
   label: Size
 ---
 
-Size refers to the restricted dimensions provided within the design system that can refer to the height and/or width of an element.
+Size refers to the restricted dimensions provided within the design system, such as an element's height and/or width.
 
 ## Size tokens
 
-| Token                    | Dimension        | High Density (HD) | Medium (MD) | Low (LD) | Touch (TD) |
+| Token                    | Dimension        | High Density (HD) | Medium      | Low      | Touch      |
```

Comment:
Why removing most but keeping HD? And shorthands are referenced in doc?

Reply:
I think not removing HD was mistake (given spacing has them all removed) but agreed we need to introduce HD/MD etc somewhere before referencing

Reply:
I'd guess something in a foundation overview, as well as SaltProvider doc

Reply:
I agree with @libertymayc. Even if we don't use those abbreviations elsewhere on the site (though, as @origami-z notes, there's a good chance we do or will at some point), they do come up in conversations between designers. It's also possible that they get used in our Figma libraries and/or code.

Consider someone who hears or stumbles across "MD" in the context of Salt and isn't sure what it means. Ideally if they search the site (once we have that implemented ;-) ) or even Google, it would be good for this page to show up in the results. Keeping all the abbreviations here would allow that to happen.